### PR TITLE
Fix upgarde typo

### PR DESCRIPTION
**Pull request by jmlrt**
NIT, fix a small typo in the `helm diff upgrade` help.

### CODE REVIEW

File: cmd/upgrade.go [hunk 1]
```diff
@@ -99,13 +99,13 @@ func newChartCommand() *cobra.Command {
...
 			"  # See https://github.com/databus23/helm-diff/issues/253 for more information.",
-			"  HELM_DIFF_USE_UPGRADE_DRY_RUN=true helm diff upgarde my-release datadog/datadog",
+			"  HELM_DIFF_USE_UPGRADE_DRY_RUN=true helm diff upgrade my-release datadog/datadog",
 			"",
 			"  # Set HELM_DIFF_THREE_WAY_MERGE=true to",
 			"  # enable the three-way-merge on diff.",
 			"  # This is equivalent to specifying the --three-way-merge flag.",
 			"  # Read the flag usage below for more information on --three-way-merge.",
-			"  HELM_DIFF_THREE_WAY_MERGE=true helm diff upgarde my-release datadog/datadog",
+			"  HELM_DIFF_THREE_WAY_MERGE=true helm diff upgeade my-release datadog/datadog",
```
Comment (bkreitch) [inline cmd/upgrade.go:108]: @jmlrt looks like there is still a typo :)

Comment (jmlrt) [reply cmd/upgrade.go]: Fixed 🤦🏻

**Review by bkreitch — APPROVED** [review]
LGTM, but I don't have any rights in this repo :)

**Comment by jmlrt** [comment]
> LGTM, but I don't have any rights in this repo :)

sorry for the ping :)

**Review by databus23 — APPROVED** [review]

**Pull request merged**